kymatio/scattering1d/filter_bank.py

```diff
@@ -405,18 +472,18 @@ def compute_xi_max(Q):
     xi_max : float
         largest frequency of the wavelet frame.
     """
-    xi_max = max(1. / (1. + math.pow(2., 3. / Q)), 0.35)
+    xi_max = max(1. / (1. + math.pow(2., 1. / Q)), 0.4)
```

Comment:
Why what? Do you refer to extending the docs beyond original? The rationale is [here](https://github.com/kymatio/kymatio/pull/764)

Reply:
@OverLordGoldDragon i still don't get why the `max` with 0.4 though. It seems like the value is `1 / (1 + 2**(1/Q))` if Q>1 and `0.4` otherwise, whereas the formula would have given `0.333`.

so the logic is:
* if Q>1, `2**(1/Q) xi = 1 - xi`
* if Q=1, `2**(1/Q) xi = 1 - xi/2`

Why does `Q=1` gets a special treatment?

Reply:
Because `0.333` is too low, we want coverage near Nyquist. Full curve for reference

<img src="https://user-images.githubusercontent.com/16495490/151013132-eb3a92ed-e876-42d1-88fe-8a350d075827.png" width="400">